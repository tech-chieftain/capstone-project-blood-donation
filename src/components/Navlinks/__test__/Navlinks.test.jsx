import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import NavLinks from '../Navlinks';

// tests will use to compare to.
it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <NavLinks
          languageflag
          setlanguageflag={() => console.log('Donate')}
          classname="hidden"
          menuClose={() => console.log('Donate')}
          classFlag={false}
        />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
