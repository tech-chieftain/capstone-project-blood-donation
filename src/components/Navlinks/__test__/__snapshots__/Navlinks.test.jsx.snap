// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ul
  className="md:flex  hidden   md:w-auto w-full text-start "
>
  <li>
    <a
      className="px-2 py-2 lg:px-6  mt-2 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white "
      href="/"
      onClick={[Function]}
    >
      nav.home
    </a>
  </li>
  <li>
    <a
      className="px-2 py-2 lg:px-6  mt-2 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white "
      href="/about"
      onClick={[Function]}
    >
      nav.about
    </a>
  </li>
  <li>
    <a
      className="px-2 py-2 lg:px-6  mt-2 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white "
      href="/instructions"
      onClick={[Function]}
    >
      nav.instructions
    </a>
  </li>
  <li>
    <a
      className="px-2 py-2 lg:px-6  mt-2 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white "
      href="/process"
      onClick={[Function]}
    >
      nav.process
    </a>
  </li>
  <li>
    <a
      className="px-2 py-2 lg:px-6  mt-2 md:mt-0 sm:mr-0 md:mr-2 lg:mr-4 text-sm  text-white tracking-wide inline-block border-b-2 border-transparent hover:border-white "
      href="/search"
      onClick={[Function]}
    >
      nav.search
    </a>
  </li>
  <li>
    <a
      className="px-4 py-2 lg:px-6 mt-2 md:mt-0  text-sm tracking-wide transition duration-500 ease-in-out bg-white inline-block text-primary transform  hover:scale-95"
      href="/donate"
      onClick={[Function]}
    >
      nav.donate
    </a>
  </li>
  <li
    className=" block  mt-4 md:mt-0 sm:mr-0 md:mr-2 order-first sm:order-first  "
  >
    <div>
      <button
        className="flex items-center"
        onClick={[Function]}
        type="button"
      >
        <img
          alt="Arabic"
          className="w-8 h-8 rounded-full object-cover object-center"
          src="arabic.svg"
        />
        <p
          className="mx-2 text-white"
        >
          عربي
        </p>
      </button>
    </div>
  </li>
</ul>
`;
